app/src/main/res/values/strings.xml

```diff
@@ -90,8 +90,7 @@
         completing the assignments.
     </string>
     <string name="correct_free_response">Your answer was saved.</string>
-    <string name="py_message">
-        <![CDATA[Please open this problem in your <a href="https://www.jetbrains.com/pycharm-educational/">PyCharm Educational Edition</a>]]></string>
+    <string name="py_message">Please open this problem in your <a href="https://www.jetbrains.com/pycharm-educational/">PyCharm Educational Edition</a></string>
```

Comment:
Text should be "Please open this problem in your JetBrains IDE", and link value `https://www.jetbrains.com/education/`.
Also resource name should follow common practices of step types related naming.